packages/synapse-react-client/package.json

```diff
@@ -38,7 +38,7 @@
     "@mui/system": "^5.14.17",
     "@mui/utils": "^5.14.17",
     "@mui/x-data-grid": "5.17.26",
-    "@mui/x-date-pickers": "^5.0.20",
+    "@mui/x-date-pickers": "^6.18.1",
```

Comment:
The `@mui/x-...` packages do not follow the same versioning as the stable MUI packages. So mixing v6 with @mui/core v5 here is ok